lab_4_fill_words_by_ngrams/main.py

```diff
@@ -39,6 +53,12 @@ def _put(self, element: str) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not (isinstance(element, str) and element):
+            raise ValueError
+        if element in self._storage:
+            return None
+        self._storage[element] = len(self._storage)
+        return None
```

Comment:
There is a better solution. `return None` is ugly

Reply:
fixed